src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -121,6 +121,11 @@ final class SSLConfiguration implements Cloneable {
     static final boolean enableDtlsResumeCookie = Utilities.getBooleanProperty(
             "jdk.tls.enableDtlsResumeCookie", true);
 
+    // Number of NewSessionTickets (NST) that will be sent by the server.
+    static final int serverNewSessionTicketCount;
+    // Default for NST
+    static final int SERVER_NST_DEFAULT = 3;
```

Comment:
I suggest you make the default 1 or 2 rather than 3.  NSTs can get pretty large and we may want to preserve bandwidth a bit, especially for servers at scale.  I think for many applications the default we're currently doing of 1 NST per connection might be enough, and the property allows folks to ratchet up the number of NSTs to fit their needs.  But defaulting to 2 is probably fine also.

Reply:
A default of 1 makes sense as it's consistent with today's NST usage.